tests/integration.rs

```diff
@@ -79,12 +79,12 @@ PROJECT_ID to be set",
     // Eat up the "connected" message
     _ = rx.recv().await.unwrap();
 
-    let project_secret = uuid::Uuid::new_v4().to_string();
+    let project_secret = std::env::var("CAST_PROJECT_SECRET").expect("CAST_PROJECT_SECRET not set");
 
     // Register project - generating subscribe topic
     let dapp_pubkey_response: serde_json::Value = http_client
         .get(format!("{}/{}/subscribe-topic", &cast_url, &project_id))
-        .bearer_auth(project_secret)
+        .header("Authorization", &project_secret)
```

Comment:
Why this change? API should be using `Authorization: Bearer <token>` to be compatible with the HTTP spec

Reply:
Yeah that's true, my bad as we did this in docs, so adjusted, but I guess we should change docs. Reverted to using bearer